app/cdap/components/GridTable/index.tsx

```diff
@@ -282,6 +296,7 @@ export default function GridTable() {
             })}
         </TableBody>
       </Table>
+      <FooterPanel recipeStepsCount={0} gridMetaInfo={tableMetaInfo} />
```

Comment:
Should `recipeStepsCount` be set by a variable?

Reply:
Currently, it is set statically but once the directives are implemented `recipeStepsCount` will be displayed from the state variable

Reply:
Currently, it is set statically but once the directives are implemented `recipeStepsCount` will be displayed from the state variable